### PR TITLE
Fix cassandra lwt

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by Arsnael** [comment]
```
[b90ab623719489a9239748aa0464d6aabaf4e92c] [ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
```
test this please

**Comment by chibenwa** [comment]
`CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution`

test this please

**Comment by chibenwa** [comment]
```
[b90ab623719489a9239748aa0464d6aabaf4e92c] [ERROR]   MemorySpamAssassinContractTest.movingAMailToTrashShouldNotImpactSpamassassinLearning » ConditionTimeout
```

test this please

**Comment by chibenwa** [comment]
Can you open a JIRA ticket for this very interesting job @mbaechler ?

File: mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraModSeqProvider.java [hunk 1]
```diff
@@ -167,7 +168,8 @@ public long highestModSeq(MailboxSession mailboxSession, MailboxId mailboxId) th
             update.bind()
                 .setUUID(MAILBOX_ID, mailboxId.asUuid())
                 .setLong(NEXT_MODSEQ, nextModSeq.getValue())
-                .setLong(MOD_SEQ_CONDITION, modSeq.getValue()))
+                .setLong(MOD_SEQ_CONDITION, modSeq.getValue())
+                .setConsistencyLevel(ConsistencyLevel.SERIAL))
```
Comment (remk) [inline mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraModSeqProvider.java]: Does the update use the read or write consistency level ?
https://docs.datastax.com/en/archived/cassandra/3.0/cassandra/dml/dmlConfigConsistency.html

I didn't find this information.
if it's the write, the serial consistency level is not listed ?

File: mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraModSeqProvider.java [hunk 2]
```diff
@@ -168,8 +169,7 @@ public long highestModSeq(MailboxSession mailboxSession, MailboxId mailboxId) th
             update.bind()
                 .setUUID(MAILBOX_ID, mailboxId.asUuid())
                 .setLong(NEXT_MODSEQ, nextModSeq.getValue())
-                .setLong(MOD_SEQ_CONDITION, modSeq.getValue())
-                .setConsistencyLevel(ConsistencyLevel.SERIAL))
+                .setLong(MOD_SEQ_CONDITION, modSeq.getValue()))
```
Comment (remk) [inline mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraModSeqProvider.java]: Same question

Comment (mbaechler) [reply mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraModSeqProvider.java]: it's actually a removal of the previous accidental insertion

**Comment by mbaechler** [comment]
It's green

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**